### PR TITLE
New site design

### PR DESCRIPTION
**Pull request by lucperkins**
This PR encompasses the "CNCF-provided" web experience for KEDA. The approach I'm taking here is very much an "ask for forgiveness and not permission." I'm going to throw something out there and project maintainers can decide what they like and don't like.

### CODE REVIEW

**Comment by lucperkins** [comment]
@tomkerkhove Could you take a look and see if this is going in a direction that you generally like?

(only the home page is templated at the moment)

**Comment by tomkerkhove** [comment]
First look is wonderful! I'll have a closer look later on but would it be possible to introduce a "Blog" section for new announcements?

**Comment by zroubalik** [comment]
I am probably stupid, but is this site rendered anywhere so I can tak a look?

**Comment by tomkerkhove** [comment]
My take on the current design.

I like this one! However, maybe it's nicer to use same color to make it nicer to read?
image

What about the subtitle "Application autoscaling made simple" and provide a more thorough what is KEDA?

Maybe we should add a white background for our circle to make it more clear? But it might lose its effect
image

I love these:
image

Would it be possible to align the sizes of these boxes or is it just me being a pain?
image

Sections I would introduce in new design:
- What is KEDA / Intro which is now hidden in concepts
- Blog to announce things

What do you think @ahmelsayed @zroubalik @jeffhollan @anirudhgarg?

**Comment by tomkerkhove** [comment]
Nice work @lucperkins, I like it a lot!

**Comment by tomkerkhove** [comment]
> I am probably stupid, but is this site rendered anywhere so I can tak a look?

That's not a stupid question @zroubalik!

You can go to the preview here: 
image

**Comment by zroubalik** [comment]
> You can go to the preview here:

@tomkerkhove Thanks!

**Comment by lucperkins** [comment]
@tomkerkhove Yep, definitely happy to provide a blog and everything else you've requested.

**Comment by tomkerkhove** [comment]
Keep going, it looks really awesome 😎

**Comment by lucperkins** [comment]
@tomkerkhove Awesome! Glad it's coming together. Please feel more than free to continue to offer suggestions, critiques, and feature requests.

**Comment by lucperkins** [comment]
@tomkerkhove Okay, this is ready for review. I did add a blog template but it's not currently linked because there are no posts. You can see what it looks like [here](https://deploy-preview-124--keda.netlify.com).

File: README.md [hunk 1]
```diff
@@ -1,24 +1,54 @@
...
+* `title`
+* `date` (in `YYYY-MM-DD` format)
+* `author`
+
+## Adding scaler documentation
+
+To add documentation for a new KEDA [scaler](https://keda.sh/docs/scalers):
+
+```sh
+hugo new --kind scaler docs/scalers/my-new-scaler.md
```
Comment (tomkerkhove) [inline README.md]: ❤️ Awesome, thanks!

**Comment by tomkerkhove** [comment]
LGTM, really like this!

A few suggestions:
- Can we make the FAQ items collapsable? Was figuring this out in #24 but removed that in favor of this PR
  - Maybe same for Troubleshooting?
- Can we move these to the same level as the other 2 titles? It feels a bit hidden now
image
- archtypes look interesting, can we use same approach for our users to allow them to easily get listed?
- I love the blog, thanks! Can we add that to the navigation?
- Since samples is an external link, I'd move that to the last entry in the navigation?

**Comment by zroubalik** [comment]
Looking really great! The boxes in `Features` section make me want to click on them, maybe it's just me?

**Comment by lucperkins** [comment]
@tomkerkhove To answer your questions:

* I can do collapsible FAQ and troubleshooting pages
* I can re-work the nav, yes
* The archetype system wouldn't quite work for adding users, but all that's required for that is updating the `config.toml`, so it's super easy. I can document that in the README.
* We can easily add the blog to the navigation by uncommenting a few lines in the `config.toml`. But I think it's better to wait until there's at least one post 😄
* I'll move samples to the end of the nav

**Comment by lucperkins** [comment]
@tomkerkhove Okay, it's ready for another look.

**Comment by tomkerkhove** [comment]
Oh man this is such a great work - Thanks!!

We have a standup on Thursday, ok if we go through it before hitting final approve?

Here are my last comments.

> Maybe there could be buttons in the main view that redirect directly to Scalers, Community sections?

This is a good idea!

> * Can we move these to the same level as the other 2 titles? It feels a bit hidden now
>   image

This is the only thing left that I would change, but not a must

> * We can easily add the blog to the navigation by uncommenting a few lines in the `config.toml`. But I think it's better to wait until there's at least one post 😄

Will PR a first post once this is merged, so feel free to add it already

**Comment by celestehorgan** [comment]
Some general comments on the homepage:

- The hero image text on the homepage is a bit difficult to read due to the background image. Consider using something like `text-shadow: 0 0 10px #000` on this to improve legibility? 
- In the Features blocks, consider aligning the icons full-left of the header + content (using them like a "bullet point" for these blocks), or centering them over each block. They look a little awkward as is.
- Maybe float the social icons in the footer full left?

Other comments:
https://deploy-preview-124--vigorous-franklin-8edd65.netlify.com/blog/ still has test posts in it that you should delete ;)

**Comment by tomkerkhove** [comment]
Last suggestion from my side since I can't PR it - Can you include this first blog post please?

/cc @zroubalik @jeffhollan 

<details>
<summary>"Kubernetes Event-driven Autoscaling (KEDA) is now an official CNCF Sandbox project 🎉" post</summary>

Source: https://docs.google.com/document/d/1WbZ7Ph687oxJMR0aHLjzTu9HfnA7EkxPXiXZo8uzKl0
```markdown
+++
title = "Kubernetes Event-driven Autoscaling (KEDA) is now an official CNCF Sandbox project 🎉"
date = 2020-03-31
author = "KEDA Maintainers"
+++

Over the past year, We've been contributing to Kubernetes Event-Driven Autoscaling (KEDA), which makes application autoscaling on Kubernetes dead simple. If you have missed it, read about it in our ["Exploring Kubernetes-based event-driven autoscaling (KEDA)"](https://blog.tomkerkhove.be/2019/06/11/a-closer-look-at-kubernetes-based-event-driven-autoscaling-keda/) blog post.

We started the KEDA project to address an essential missing feature in the Kubernetes autoscaling story. Namely, the ability to autoscale on arbitrary metrics. Before KEDA, users were only able to autoscale based on metrics such as memory and CPU usage. While these values are essential for autoscaling, they disregard a rich world of external metrics from sources such as Azure, AWS, GCP, Redis, and Kafka (among many more).

To address this need, KEDA provides a simple, unified API to autoscale deployments without  an in-depth knowledge of Kubernetes internals. With KEDA, users can now treat their Kubernetes deployments like FaaS or PaaS applications with ease!

In the incredible year since we announced KEDA publicly, adoption has been increasing, and every week we find more passionate and excited members in our weekly community standups. Members of the Kubernetes community have been incredibly accepting. They have been providing feedback, contributing features, and offering great suggestions for the future of our project.

On November 19, 2019, we released [Kubernetes Event-Driven Autoscaling (KEDA) v1.0](https://cloudblogs.microsoft.com/opensource/2019/11/19/keda-1-0-release-kubernetes-based-event-driven-autoscaling/). This release introduced a ton of features including support for multiple workloads (deployments & jobs), simplified deployment with Helm, documentation on [keda.sh](http://keda.sh/), and (my personal favorite) enterprise-class security with TriggerAuthentication CRD (which allows you to use pod identities such as Azure Managed Identity for pods).

Over time our community has grown - More and more companies such as IBM, Pivotal, VMware, [Astronomer](https://www.astronomer.io/),  and more started contributing to KEDA, we are collaborating with [Knative project](https://knative.dev/) to provide seamless integration with each other and our user base started growing with companies such as, [Purefacts](https://www.purefacts.com/), [SwissRe](https://www.swissre.com/) and more!

We want to give KEDA more room to grow independently and ensure it has a vendor-agnostic focus. That's why on Jan 14, 2020, we proposed KEDA to  the CNCF as a new Sandbox project.

**Today, we are happy to announce that KEDA is now an [official CNCF Sandbox project](https://www.cncf.io/sandbox-projects/)!** By contributing KEDA to the CNCF we hope to ensure the adoption of KEDA continues to increase and hope to see more companies contribute scalers, integrate it in their products and give it a neutral home. This is a major step and I'm sure the best is yet to come.

We would love to explicitly thank Liz Rice, Michelle Noorali & Xiang Lifor being our CNCF TOC sponsors and supporting KEDA as well as SIG-Runtime, especially Ricardo Aravena, for recommending us to TOC!

So... what's next?

In the near-term, we plan to focus on two major topics:  Autoscaling HTTP workloads and scalers!

Currently, we do not support HTTP-based autoscaling out-of-the-box, so we hope to create on a Service Mesh Interface (SMI) scaler for autoscaling service mesh workloads!

In parallel, we have started plans for implementing add-on scalers. What are add-on scalers? We’re glad you've asked! Add-on scalers make it easy for users to define custom [external scalers](https://keda.sh/scalers/external/) without needing to contribute code to KEDA directly. One example of an external scaler is the [Azure Durable Function scaler](https://github.com/kedacore/keda-scaler-durable-functions).

As this project evolves, our main focus will be to provide guidelines around when to add a scaler to the core and when  to offer it as an external add-on. Next to that, we can create a centralized hub for all add-on scalers to improve discoverability similar to what [Helm Hub](https://hub.helm.sh/) provides.

We have a lot of ideas and plans but we mainly are interested in what you want! Are you missing scalers, features or capabilities? Let us know!

Thanks for reading, and happy scaling!

KEDA Maintainers.
```
</details>

**Comment by lucperkins** [comment]
@tomkerkhove Done

**Comment by tomkerkhove** [comment]
You are fast, Sir - Thanks!

**Comment by tomkerkhove** [comment]
If you are ready I think we can merge @lucperkins?

We have a standup on Thursday where we'll discuss this but feel better to request changes when we have them unless you prefer to keep it open until it's reviewed?

**Comment by zroubalik** [comment]
@lucperkins looking really good! I have minor comments:
In the docs sections, almost on every "main page" such as, `Deploying KEDA`, `FAQ`, `Troubleshooting`, etc there is `Page Source` piece, maybe it is just wrongly redendered in the preview and the page is ok? See the screenshot.

Page Source


And the second thing, there are currently 2 `Community` sections with slightly different content and appearance, one on the main page in the bottom (I like the look & feel of this one), second accessible throught the buttons on the top of the page.

**Comment by zroubalik** [comment]
@tomkerkhove do you happen to know what is the status of the improved logo? (the one with the explicit thunder) I like it more, and would love to see it on the new web page. Thanks!

**Comment by tomkerkhove** [comment]
> @lucperkins looking really good! I have minor comments:
> In the docs sections, almost on every "main page" such as, `Deploying KEDA`, `FAQ`, `Troubleshooting`, etc there is `Page Source` piece, maybe it is just wrongly redendered in the preview and the page is ok? See the screenshot.

What's your concern here, that it's not nicely formatted or the goal? Think it's easier to discover it, but now that I think of it `Suggest a change 📝` might make more sense then?

**Comment by tomkerkhove** [comment]
@zroubalik we're not there yet unfortunately (https://github.com/kedacore/keda/issues/650) but should be easy to change afterwards I think?

**Comment by tomkerkhove** [comment]
Another minor suggestion is "Resources" or "Further reading".

On landing page we have this:
image

But actually we have more content now (https://github.com/kedacore/keda-docs/issues/113) so would be good if we could have a page where we can group them.

**Comment by zroubalik** [comment]
> > @lucperkins looking really good! I have minor comments:
> > In the docs sections, almost on every "main page" such as, `Deploying KEDA`, `FAQ`, `Troubleshooting`, etc there is `Page Source` piece, maybe it is just wrongly redendered in the preview and the page is ok? See the screenshot.
> 
> What's your concern here, that it's not nicely formatted or the goal? Think it's easier to discover it, but now that I think of it `Suggest a change 📝` might make more sense then?

@tomkerkhove besically both. I don't like the formatting and wasn't sure what was the meaning of that.  `Suggest a change 📝` make more sense for me, I like it!

**Comment by lucperkins** [comment]
@tomkerkhove Since the blog post is for today, probably best to merge now and follow up.

**Comment by tomkerkhove** [comment]
Ah the date is just an indication, no worries.

Feel free to have a look at @zroubalik his comments as well if you want.

**Comment by lucperkins** [comment]
Okay, I think I've addressed the existing concerns.

**Comment by zroubalik** [comment]
@lucperkins looking really great, I like the drop down menu on Docs. I have found 2 minor issues:

1. `Resources` button in the menu doesn't work for me.
2. The Community section is missing information about weekly call etc. Would be great to add button that redirects to that section.

**Comment by lucperkins** [comment]
@tomkerkhove Done

**Comment by lucperkins** [comment]
@zroubalik It's up to you, ultimately. But I would recommend keeping the separate page (right now, the material from that page is automatically piped to the **Get Involved** section on the main page). Community pages have a tendency to grow over time. You may find yourself with something like this a year down the road: https://prometheus.io/community.

**Comment by zroubalik** [comment]
@lucperkins makes sense! Agree. Thanks, we might want to add the Users section to the top of the separate Community page as well

**Comment by lucperkins** [comment]
@zroubalik Okay, sounds good! Feel free to file an issue and assign me at some point if you want to take that route.

**Comment by tomkerkhove** [comment]
I'll pull the trigger on this one, awesome work on this @lucperkins - Thanks so much!

**Pull request merged**